README.md

```diff
@@ -124,11 +122,19 @@ Recommended for use with `onChange`. Will be auto-generated if not provided.
 
 #### className : `string` [*optional*, default: `'accordion__heading'`]
 
-Class(es) to apply to element.
+Class(es) to apply to the 'heading' element.
 
-#### expandedClassName : `string` [*optional*, default: `'accordion__heading--expanded'`]
+#### aria-level : `number` [*optional*, default: `3`]
 
-Class(es) to append when item is expanded.
+Semantics to apply to the 'heading' element. A value of `1` would make your
+heading element hierarchically equivalent to an `<h1>` tag, and likewise a value
+of `6` would make it equivalent to an `<h6>` tag.
+
+### AccordionItemButton
+
+#### className : `string` [*optional*, default: `'accordion__heading'`]
```

Comment:
Think you maybe forgot to update the class here?

Reply:
Well spotted, cheers.